docker/Dockerfile-debian

```diff
@@ -50,7 +50,7 @@ COPY ./poetry.lock /trezor-user-env/
 RUN poetry self add poetry-plugin-export
 RUN poetry export -f requirements.txt --output requirements.txt --without-hashes --without dev
 RUN pip install -r requirements.txt
-RUN pip install "git+https://github.com/trezor/trezor-firmware.git@022c7fe8bee9bd373c724bf88f618a75658de625#egg=trezor&subdirectory=python"
+RUN pip install "git+https://github.com/trezor/trezor-firmware.git@3daedcc7778daa5fd1242b378f68a9407110f5d8#egg=trezor&subdirectory=python"
```

Comment:
I basically need to force a new image so to achieve this I decided to update python trezor-version